### ISSUE TITLE
spa

### ISSUE THREAD

**Issue by ericxin1982** [imacrayon/alpine-ajax#17]
Hi @imacrayon 

Sorry for another question, I am too new to Alpine.

I used to use vue3, and I am tired of doing spc and webpack/vite, I have to use vue3 since SPA.
Alpine seems very similar to vue, but I am not sure how to use Alpine for spa way.

Would you please give me some instructions?

Thanks
Eric Xin

**Comment by imacrayon** [comment]
No worries. Alpine is designed to work with server-rendered apps, it doesn’t work very well with SPAs. That’s why I built Alpine AJAX, so that server-rendered apps using Alpine can still feel like a SPA. 
Typically, Vue will consume JSON sent from the server. Alpine works best with HTML shipped from the server instead.
You’ll need to pair Alpine with a server-side language to build apps, what framework are you using on the server to build your APIs for Vue?

**Comment by ericxin1982** [comment]
I am using Powershell pode.

I saw alpine history knew its author why made it, it consumes html totally instead of json, that is cool, that is new to me. 
But that is familiar with the way I use to work, I can use alpine-ajax you built to download html template from server A, and download json from server B, and assembly data and template on client side by alpine, that is what I did before in vue1, that is faster without webpack involved. (That is why I need some time delay between after download html template, and assembly data to it).

I rather use html template with json on client side, I can get benefit from server rendering and enough flexibility, but some secure content can be html with data from server for sure, your product gives me more choice.

That is why I asked you what UI you are using. I just plan to use comprehensive UI solution for template design.
Thanks for your great help!

Thanks
Eric Xin
